proto-google-cloud-talent-v4beta1/clirr-ignored-differences.xml

```diff
@@ -16,4 +16,13 @@
     <className>com/google/cloud/talent/v4beta1/*OrBuilder</className>
     <method>boolean has*(*)</method>
   </difference>
+  <difference>
+    <differenceType>8001</differenceType>
+    <className>com/google/cloud/talent/v4beta1/*Name*</className>
```

Comment:
Let's be more explicit on exactly which classes are ok to remove in this PR.

Reply:
done